### PR TITLE
feat: use 'Github Actions' to run tests and auto release

### PR DESCRIPTION
**Pull request by basakest**

### CODE REVIEW

**Comment by hsluoyz** [comment]
@leeqvip

File: composer.json [hunk 1]
```diff
@@ -33,8 +33,7 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.0",
-        "php-coveralls/php-coveralls": "^2.1"
+        "phpunit/phpunit": "^7.0"
     },
```
Comment (leeqvip) [inline composer.json]: Why remove php-coveralls ?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          MYSQL_DATABASE: casbin
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [ 7.2, 7.3 ]
```
Comment (leeqvip) [inline .github/workflows/build.yml]: Please add php7.4、8.0.

**Comment by UtkarshaAnimationMedia** [comment]
sdfghjkl;

File: composer.json [hunk 3]
```diff
@@ -33,8 +33,7 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.0",
-        "php-coveralls/php-coveralls": "^2.1"
+        "phpunit/phpunit": "^7.0"
```
Comment (leeqvip) [inline composer.json]:
```suggestion
        "phpunit/phpunit": "^7.0",
        "php-coveralls/php-coveralls": "^2.1"
```

File: .github/workflows/build.yml (hunk 2 above)
Comment (leeqvip) [inline .github/workflows/build.yml]:
```suggestion
        php: [ 7.2, 7.3,7.4,8.0 ]

```

**Pull request merged**